medical_medicament_us/models/medical_medicament.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class MedicalMedicament(models.Model):
+    _inherit = 'medical.medicament'
+
+    ndc_ids = fields.One2many(
+        string='NDCs',
+        comodel_name='medical.medicament.ndc',
+        inverse_name='medicament_id',
+        ondelete='cascade',
+        help='National Drug Codes for medicament.',
+    )
+    gcn_id = fields.Many2one(
+        string='GCN',
+        comodel_name='medical.medicament.gcn',
+        help='Generic Code Number for medicament.',
+    )
+    gpi = fields.Selection(
+        string='GPI',
+        selection=[
+            ('0', '0 (non-drug)'),
+            ('1', '1 (generic)'),
+            ('2', '2 (brand)'),
+        ],
+        default='0',
+        help='Generic Product Identifier for medicament.',
+    )
+    control_code = fields.Selection(
+        string='Control Code',
+        selection=[
+            ('0', 'Not Controlled'),
+            ('1', 'C1'),
+            ('2', 'C2'),
+            ('3', 'C3'),
+            ('4', 'C4'),
+            ('5', 'C5'),
+        ],
+        help='Federal drug scheduling code for medicament.',
```

Comment:
http://medshadow.org/resource/drug-classifications-schedule-ii-iii-iv-v/

Think it might be worth it to add quick help description of the codes. @lasley do you think that might be worth it or just leave it?